web-client/public/js/setup-handlers.js

```diff
@@ -239,8 +239,8 @@ export const setupHandlers = grnState => {
         updateApp(grnState);
     });
 
-    $(TOP_DATASET_SELECTION_MENU).click(() => {
-        var selection = $(this).attr("value");
+    $(TOP_DATASET_SELECTION_MENU).click((event) => {
+        const selection = event.target.textContent.trim();
```

Comment:
This was reviewed at the meeting and it was indicated that `event.target` is an `a` element because the selection happens in the menu. As such, the preferred solution here is to use a `data` attribute (https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes)—we will need to find the code that generates these menu items (they must be dynamic because they are derived from the database and/or the opened workbook) and revise the menu items to include the `data` attribute (name tbd)

Once we have those data attributes, then we will have a more robust way to identify the chosen expression data set in this code (i.e., it becomes `event.target.dataset[something-name-tbd]`